# goal
float64 number
---
# result
bool status
---
# feedback
float64 progress
